model/common/src/icon4py/model/common/decomposition/definitions.py

```diff
@@ -59,21 +59,37 @@ def __call__(self):
             self._counter = self._counter + 1
         return next_id
 
+@dataclasses.dataclass(frozen = True)
+class MaskedArray:
+    data: np.ndarray
+    mask: np.ndarray
+    
+    def __post__init__(self, ):
```

Comment:
It looks like it has an extra _ between post and init

Reply:
thanks